core/kazoo_amqp/src/api/kz_api.erl

```diff
@@ -63,6 +63,21 @@
         ,validate_message/4
         ]).
 
+%% #kapi_definition accessors
```

Comment:
I would think a kapi_definition module would be a good place to put this stuff (and the record definition).